README.md

```diff
@@ -137,6 +137,18 @@ const context = async ({ req }) => {
 };
 ```
 
+## Back-End Testing
+
+To run tests, cd into the apollo directory and run 'npm test' on your terminal.
+
```

Comment:
I would mention also running 
`npm i`
Just incase they're like me and forget to 😅

Reply:
Also since I already had a prisma environment set up I had to run the following sequence to get the test to NOT fail....because of the schema/seed data...

`prisma delete`
`prisma generate`
`prisma deploy`

This will delete whats currently living in prisma, then it will generate the "new" data and then will deploy.